comm/src/test/scala/coop/rchain/comm/transport/TransportLayerSpec.scala

```diff
@@ -32,7 +32,7 @@ abstract class TransportLayerSpec[F[_]: Monad: cats.effect.Timer, E <: Environme
 
           protocolDispatcher.received should have length 1
           val (_, protocol2)           = protocolDispatcher.received.head
-          val sender: Option[PeerNode] = ProtocolHelper.sender(protocol2)
+          val sender: Option[PeerNode] = Some(ProtocolHelper.sender(protocol2))
```

Comment:
This can be defined without Option, so that only _sender_ is compared to _result.localNode_.

Reply:
Done